### DISCUSSION TITLE
How to display the whole map, all tiles available in one image

### DISCUSSION THREAD

**Discussion by boeckMt** [openlayers/openlayers#14838] [Q&A] [answered]
I have a use case where I want to display a world map next to a chart in a smaller region (The chart layer has been omitted for simplicity), but the full map should be displayed without panning the map. Currently, if the map size is not a square on `EPSG:3857`, things are cut off, but I want to fit the whole map into the container size.

Maybe I could use a different projection so that I do not need a square size to view the full map.
Another option would be to place a squaresized map in a container, and use CSS to position the map in the centre of the container.

Example:

| currently |  needed|
|-----------|----------|
| ![Unbenannt1](https://github.com/openlayers/openlayers/assets/3969484/4510d80e-9aca-418f-8807-f966619b7ceb)| ![Unbenannt2](https://github.com/openlayers/openlayers/assets/3969484/b9370b83-1e4a-4bba-82fc-2af6bfd75882)|

Or is there a way to zoom out more than zoom 0 so that the full map is displayed in the map container?

**Comment by jahow** [comment, marked as answer]
You can set `multiWorld: false` when creating the View, see https://openlayers.org/en/latest/apidoc/module-ol_View-View.html

By default this is set to `true` and prevents zooming out too much.

**Reply by boeckMt** [reply]
@jahow, thanks I don't rememberd there was such an option, only looked for `wrapX` of `Source`.

https://github.com/openlayers/openlayers/blob/03dae4fe9bf45dd86b56dcce400dc2f2da49cf26/src/ol/View.js#L1973
I get the desired result by creating the view with `multiWorld: true' (looks like default is false).
